engine/lite/embeddedDeploymentManager/src/test/scala/pl/touk/nussknacker/streaming/embedded/StreamingEmbeddedDeploymentManagerTest.scala

```diff
@@ -72,84 +72,6 @@ class StreamingEmbeddedDeploymentManagerTest
     manager.getScenarioDeploymentsStatuses(name).futureValue.value shouldBe List.empty
   }
 
-  test("Run persisted scenario deployments") {
```

Comment:
It is not so easy to test embedded DM after removing `ProcessingTypeDeployedScenariosProvider`. To do this, we should probably add some separated test for this DM inside designer module. I think that it is not a priority now, so I only tested if it works manually.

Reply:
Maybe should we leave todo somewhere?